### PR TITLE
Fixing vagueness on the article.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Fixing Issue #2627

### CODE REVIEW

**Comment by scanum** [comment]
v

File: Skype/SfBServer2019/plan/system-requirements.md [hunk 1]
```diff
@@ -535,7 +535,10 @@ Skype for Business Server 2019 can use the same file share for all file storage.
...
     
-- We recommend a shared cluster for the file share. If you're using one, you should cluster Windows Server 2012 or Windows Server 2012 R2. Why the latest Windows? Older versions may not have the right permissions to enable all features. You can use Cluster Administrator to create the file shares, and this [Creating a Cluster](https://support.microsoft.com/en-us/help/224967) KB article will help you with those details.
+- We recommend a shared cluster for the file share. If you're already using one, you should cluster Windows Server 2012 or higher versions
+
+> [!Note]
+> **Why the latest Windows?** Older versions may not have the right permissions to enable all features. You can use Cluster Administrator to create the file shares, and this [Creating a Cluster](https://support.microsoft.com/en-us/help/224967) KB article will help you with those details.
```
Comment (nenonix) [inline Skype/SfBServer2019/plan/system-requirements.md]:
```suggestion
> **Why the latest Windows?** Older versions may not have the right permissions to enable all features. You can use Cluster Administrator to create the file shares. See this support article [How to create file shares on a cluster](https://support.microsoft.com/help/224967) for more details.
```

**Comment by j0rt3g4** [comment]
Added the polishing  from @nenonix

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**